packages/@aws-cdk/pipelines/lib/pipeline.ts

```diff
@@ -112,10 +157,19 @@ export class CdkPipeline extends Construct {
    * You can use this to add more Stages to the pipeline, or Actions
    * to Stages.
    */
-  public get pipeline(): codepipeline.Pipeline {
+  public get codePipeline(): codepipeline.Pipeline {
```

Comment:
call out as a breaking change?

Reply:
This sucks. The previous one was committed by accident. I believe there hasn't been a release in between.